.github/workflows/run-test-cases.yml

```diff
@@ -179,9 +179,6 @@ jobs:
         sudo microk8s.enable helm3
         sudo microk8s.enable rbac
         sudo microk8s.enable dns
-        echo "--allow-privileged=true" | sudo tee -a /var/snap/microk8s/current/args/kube-apiserver
-        sudo microk8s.stop
-        sudo microk8s.start
```

Comment:
is the restart needed for enabling rbac or dns or helm3?

Reply:
No. I dont think it is. MicroK8s docs says "enable add-ons" and then doesnt specify stop and restart: https://microk8s.io/docs

Reply:
👍